src/components/CharacterInfo.vue

```diff
@@ -1,71 +1,81 @@
+<script setup>
+import CharacterInfoInput from "./CharacterInfoInput.vue";
+import CharacterInfoSelect from "./CharacterInfoSelect.vue";
+import CharacterInfoPortrait from "./CharacterInfoPortrait.vue";
+</script>
+
 <script>
 /**
- * Displays inputs for name and age
- * displays selects for clan, generation, faction
+ * Displays basic character info in inputs or selects,
+ * emits detected changes upwards to have data fresh for export or potential other work
  */
 export default {
-  props: ["bio", "clans"],
+  props: ["bio"],
+  emits: ["bioChange"]
 };
 </script>
 
 <template>
-  <div class="char-info">
-    <input
-      class="heading1"
-      type="text"
-      v-model="bio.name"
-      :placeholder="bio.defaultName"
-    />
-    <b>Age:</b>
-    <input
-      :style="{ width: 7 + 'ch' }"
-      v-model="bio.age"
-      type="number"
-      :placeholder="123"
-    />
-    <b>Generation:</b>
-    <select v-model="bio.generation">
-      <option>15th</option>
-      <option>14th</option>
-      <option>13th</option>
-    </select>
-    <b>Clan:</b>
-    <select v-model="bio.clan">
-      <option disabled value="">Pick a Clan</option>
-      <option v-for="clan in clans" :key="clan.id" :value="clan">
-        {{ clan.id }}
-      </option>
-    </select>
-    <b>Faction:</b>
-    <select v-model="bio.faction">
-      <option disabled value="">Choose Your Coterie's Faction</option>
-      <option value="Anarch">Anarch</option>
-      <option value="Autarki">Autarki</option>
-      <option value="Camarilla">Camarilla</option>
-      <option value="Sabbat">Sabbat</option>
-    </select>
+  <div class="char-info" id="char-info">
+    <div class="char-info-group">
+      <CharacterInfoInput 
+        :item="bio.concept" 
+        @char-item-change="$emit('bioChange', [bio.concept, $event])"></CharacterInfoInput>
```

Comment:
tu sa da dat 
 
v-model:item="bio.concept"

miesto

:item="bio.concept"

ale vlastne je to len dlhsie, lebo sa to aj tak musi emitovat vyssie a ten emit tam musi nechat. Ked ho dam prec rozbiju sa discipliny

Reply:
rozbiju sa discipliny? ako to?

Reply:
ked dam prec event @char-item-change tak sa nic neposle hore a neurobi sa zmena klanu, takze disciplinu su furt na thin blood. Takze si mozem vybrat, ci tam bude bez v-model: 

`      <CharacterInfoSelect 
        :item="bio.clan"
        @char-item-change="$emit('bioChange', [bio.clan, $event])"></CharacterInfoSelect>`

alebo s v-model

      <CharacterInfoSelect 
        v-model:item="bio.clan"
        @char-item-change="$emit('bioChange', [bio.clan, $event])"></CharacterInfoSelect>

Reply:
uf to je divne, lebo v main je
`<select v-model="bio.clan">
      <option disabled value="">Pick a Clan</option>
      <option v-for="clan in clans" :key="clan.id" :value="clan">
        {{ clan.id }}
      </option>
    </select>`
    a tak uplne nerozumiem co sa deje

Reply:
co ja viem, mozno som to nezkumal dost... :-)